x-pack/plugins/screenshotting/README.md

```diff
@@ -3,18 +3,153 @@
+import type { ScreenshottingStart } from 'x-pack/plugins/screenshotting/server';
+
+
+interface StartDeps {
+  screenshotting: ScreenshottingStart;
+}
+
+class ExamplePlugin implements Plugin<void, void, void, StartDeps> {
+  setup({ http, getStartServices }: CoreSetup<StartDeps>) {
+    const router = http.createRouter();
+
+    router.get(
+      {
+        path: '/api/capture',
+        validate: {
+          query: schema.object({
+            id: schema.string(),
+          }),
+        },
+      },
+      async (context, request, response) => {
+        const [, { screenshotting }] = await getStartServices();
+        const { metrics, results } = await screenshotting
+          .getScreenshots({
+            request,
+            urls: [`http://localhost/app/canvas#/workpad/workpad-${request.query.id}`],
+          })
+          .toPromise();
+
+        return response.ok({
+          body: JSON.stringify({
+            metrics,
+            image: results[0]?.screenshots[0]?.data.toString('base64'),
+            errors: results[0]?.renderErrors,
+          } as ScreenshottingExpressionResponse),
+        });
+      }
+    );
+  }
+
+  start() {}
+}
+
+export function plugin() {
+  return new ExamplePlugin();
+}
+```
+
+### API
+Please use automatically generated API reference or generated TypeDoc comments to find the complete documentation.
+
+#### `getScreenshots(options): Observable`
+Takes screenshots of multiple pages or an expression and returns an observable with the screenshotting results.
 
+The `options` parameter is an object with parameters of the screenshotting session.
+Option | Required | Default | Description
+--- | :---: | --- | ---
+`browserTimezone` | no | _none_ | The browser timezone that will be emulated in the browser instance. This option should be used to keep timezone on server and client in sync.
+`expression` | no | _none_ | An expression to capture screenshot of. Mutually exclusive with the `urls` parameter.
+`format` | no | `'png'` | An output format. It can either be PDF or PNG. In case of capturing multiple URLs, all the screenshots will be combined into one document for PDF format. For PNG format, an array of screenshots will be returned.
+`headers` | no | _none_ | Custom headers to be sent with each request. The headers will be used for authorization.
+`input` | no | `undefined` | The expression input.
+`layout` | no | `{}` | Page layout parameters describing characteristics of the capturing screenshot (e.g., dimensions, zoom, etc.).
+`request` | no | _none_ | Kibana Request reference to extract headers from.
+`timeouts` | no | _none_ | Timeouts for each phase of the screenshot.
+`timeouts.loadDelay` | no | `3000` | The amount of time in milliseconds before taking a screenshot when visualizations are not evented. All visualizations that ship with Kibana are evented, so this setting should not have much effect. If you are seeing empty images instead of visualizations, try increasing this value.
```

Comment:
In truth, this value is not used at all. It used on a condition that is never true: https://github.com/elastic/kibana/issues/123514

I guess we added this to the documentation (https://github.com/elastic/kibana/issues/123514), but now we need to follow up with an issue to remove this property from the code. 

Filed: https://github.com/elastic/kibana/issues/129967